protobuf/sync/v1/sync_service.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+import "google/rpc/status.proto";
+
 package sync.v1;
```

Comment:
Given the breaking change, I suggest bumping the version
```suggestion
package sync.v2;
```

Reply:
Thanks, yes this makes sense. 

I will first wait for more discussions with the project topics - https://github.com/orgs/open-feature/projects/8 . I see there will be more changes coming to grpc contract soon.